packages/ruby/spec/readme/payload_spec.rb

```diff
@@ -5,7 +5,20 @@
   it "returns JSON matching the readmeMetrics schema" do
     har_json = File.read(File.expand_path("../../fixtures/har.json", __FILE__))
     har = double("har", to_json: har_json)
-    result = Readme::Payload.new(har)
+
+    env = {
+      "current_user" => {"id" => "1", "name" => "Anthony", "email" => "anthony@exampel.com"}
+    }
+
+    get_user_info = proc { |env|
+      {
+        id: env["current_user"]["id"],
+        label: env["current_user"]["name"],
+        email: env["current_user"]["email"]
+      }
+    }
+
+    result = Readme::Payload.new(har, get_user_info.call(env))
```

Comment:
Thoughts on using a block here directly instead of a proc to better mirror real usage?

Reply:
I think this was changed before you commented. I decided that it was closer to real usage to just pass the value in, as that's what happens in the `Readme::Metrics` class. How's this look now?